### PR TITLE
GIT-49 Implement results component

### PR DESCRIPTION
**Pull request by Virtover**

### CODE REVIEW

**Review by Virtover — COMMENTED** [review]
pre-commit.ci autofix

File: frontend/src/components/Results.js [hunk 1]
```diff
@@ -1,5 +1,82 @@
...
+          setError(true);
+        }
+      } catch (error) {
+        setError(true);
+      }
+    };
+
+    const fetchRepository = async () => {
+      try {
+        const response = await fetch(`${BACKEND_URL}/repository/${state.gameId}/`);
```
Comment (lukaszsmolinski) [inline frontend/src/components/Results.js]: The id of the repository, not the game, should be used here.

File: frontend/src/components/Results.js [hunk 2]
```diff
@@ -1,5 +1,89 @@
...
+        }
+      } catch (error) {
+        setError(true);
+      }
+    };
+
+    const fetchRepository = async () => {
+      try {
+        const response = await fetch(
+          `${BACKEND_URL}/repository/${state.gameId}/`
```
Comment (lukaszsmolinski) [inline frontend/src/components/Results.js:33]: I think that the id of the repository, not the game, should be used here.

File: frontend/src/components/Results.js
Comment (lukaszsmolinski) [inline frontend/src/components/Results.js:1]: Can you also make it possible to go back to the home screen?

**Review by lukaszsmolinski — CHANGES_REQUESTED** [review]

**Review by lukaszsmolinski — APPROVED** [review]

**Review by kate20031 — APPROVED** [review]

**Review by zemdam — APPROVED** [review]

**Pull request merged**